examples/apps/generic_client/generic_client.py

```diff
@@ -165,9 +171,6 @@ def Main(args=None):
     logger.info("***************** TRUSTED COMPUTE FRAMEWORK (TCF)" +
                 " *****************")
 
-    global direct_jrpc
-    direct_jrpc = AvalonDirectClient(config_file=None, config=config)
-
     global address
```

Comment:
why do we have so many global variables defined. This way of writing code is not good at all. It looks that demo.py has been referenced while developing this file. Please take action on this cleanup. This is my comment. Seperate PR can be raised for that cleanup. I'm not making this as blocking for this PR.

Reply:
Yes. This issue was identified earlier also. This can be fixed as a separate PR. I'll raise an issue in github

Reply:
I addressed this in this PR itself